src/gui/torrentcategorydialog.h

```diff
@@ -55,4 +56,5 @@ class TorrentCategoryDialog : public QDialog
 
 private:
     Ui::TorrentCategoryDialog *m_ui;
+    QPushButton *m_saveButton;
```

Comment:
The changes in this file seems not in use, please revert them.

Reply:
Hmm this file seems to have been reverted to previous. I'll update it again. Thanks!

Reply:
Deleted